prereise/gather/hiflddata/plant_agg.py

```diff
@@ -0,0 +1,211 @@
+import csv
+
+import pandas as pd
+from geopy.distance import geodesic
+
+from prereise.gather.hiflddata.data_trans import Clean, get_Zone
+
+Max_Value = 3000
+Min_Value = 0
```

Comment:
It seems that these values are not used in this module?

Reply:
will remove them in next revision.